scrapers/wy/events.py

```diff
@@ -88,7 +89,12 @@ def scrape(self):
                         bills_agenda_item = event.add_agenda_item(
                             "Bills under Consideration"
                         )
-                    bills_agenda_item.add_bill(bill["billNumber"])
+                    # Separate "SF" from "0012" in "SF0012" to strip leading zeroes
+                    num_match = re.search('([A-Z]+)(\d+)',bill["billNumber"] )
```

Comment:
Have to add an `r` before the regex string so it knows it's regex. Also space after comma in parameters & not before closing parentheses